articles/android-recyclerviews/index.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/android-recyclerviews/
+title: Android Recyclerviews
+description: This article goes through
+author: linus-muema
+date: 2020-11-04T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/android-recyclerviews/hero.jpg
+    alt: Android recyclerviews image
+---
+Loading vast amounts of data into normal views can lead to high CPU usage. This, in turn, may lead to the operating system forcibly shutting down the applications.
```

Comment:
please go over the first few paragraphs as I moved a few things around and added some things. I want to make sure I didn't take out anything important or change the meaning so much that is says something different.

Reply:
I was aiming to start with a short fact about receiving data in android. Then I would lead the reader to how bad management of a lot of data can be an awful practice. Then have a short history of what has been done before to help the issue until we get to the `recyclerView` point of the article. But it is still okay as it is.